core/templates/dev/head/pages/exploration_player/ExtractImageFilenamesFromStateService.js

```diff
@@ -123,27 +123,23 @@ oppia.factory('ExtractImageFilenamesFromStateService', [
       var imageTagList = dummyElement.getElementsByTagName(
         'oppia-noninteractive-image');
       for (i = 0; i < imageTagList.length; i++) {
-        var filepathObject = JSON.parse(
+        var filename = JSON.parse(
           imageTagList[i].getAttribute('filepath-with-value'));
         // The images already there in Oppia have image filenames as the value
-        // for the attribute 'filepath-with-value'. In explorations the
-        // the attribute value is an object of the form --
+        // for the attribute 'filepath-with-value'. We later on want to have
+        // it's value as the following object of the form --
```

Comment:
it's --> its

That said, I don't think you need the part of the comment from "We later on ..." -- or this comment at all. In fact something seems wrong with this entire method because it's talking about a "filepath object" which simply doesn't exist at the moment. Can it be removed?

Reply:
Yes, will remove it.

Reply:
Done!